docs/src/app/app.jsx

```diff
@@ -1,20 +1,37 @@
 import './app.scss';
 
-import _ from 'lodash';
-import { Button, domUtils, Icon } from 'react-cm-ui';
-import { Link } from 'react-router';
+import {
+    AppBar,
+    Button,
+    domUtils,
+    Icon,
+    Typography,
+} from 'react-cm-ui';
+import { Link, withRouter } from 'react-router';
+import _, {
```

Comment:
Can we remove the underscore import? were only using a `_.debounce` in line 70.

Reply:
Fixed.